code-samples/miscellaneous/parsing-mention-arguments/11/index.js

```diff
@@ -54,6 +54,26 @@ client.on('message', message => {
 
 		return message.channel.send(`${message.author.username}, your avatar: ${message.author.displayAvatarURL}`);
 	}
+
+	if (command === 'ban') {
+		if (args.length < 2) {
+			return message.reply('Please mention the user you want to ban and specify a ban reason.');
+		}
+
+		const user = getUserFromMention(args[0]);
+		if (!user) {
+			return message.reply('Please use a proper mention if you want to ban someone.');
+		}
+
+		const reason = args.slice(1).join(' ');
+		try {
+			await message.guild.ban(user, { reason });
+		} catch (e) {
```

Comment:
`catch (e)` should be `catch (error)` in each of these, to stay consistent with other guide pages.